src/mlpack/bindings/cli/map_parameter_name.hpp

```diff
@@ -67,7 +67,7 @@ void MapParameterName(util::ParamData& d,
   // Store the mapped name in the output pointer, which is actually a string
   // pointer.
   *((std::string*) output) =
-      MapParameterName<typename std::remove_pointer<T>::type>(d.name);
+      MapParameterName<std::remove_pointer_t<T>>(d.name);
```

Comment:
I don't think so, it will end at 81 > 80 columns.

Reply:
Sorry about that, I see I got a few of these wrong.  For some of them I was just eyeballing it and did not 100% check (but I was pretty sure it would be shorter!  I guess not).

Reply:
No problem, I will go through all your findings and also found already a few additional places, where I could follow your guidelines better. I just answer to all the comments in order to keep some kind of overview :wink:.